x/protorev/types/genesis.go

```diff
@@ -20,6 +20,10 @@ var (
 		StableWeight:       5, // it takes around 5 ms to simulate and execute a stable swap
 		BalancerWeight:     2, // it takes around 2 ms to simulate and execute a balancer swap
 		ConcentratedWeight: 2, // it takes around 2 ms to simulate and execute a concentrated swap
```

Comment:
Since in upgrade we are resetting pool weights to the defaults, can we match the defaults to what's on mainnet today? (BalancerWeight is 1, ConcentratedWeight is 300 -- essentially turning it off)

Reply:
The CL issue should go away in the upgrade right?

Reply:
Yes! There is an outstanding task (which we will open a PR for in the next few days) that will bound protorev search better for CL pools and allow it to work on mainnet.

Reply:
I'll retroactively update the pool weights in the update PR.

Reply:
@ValarDragon Yeah the issue will go away, but I'd like to change the pool points in the PR that makes the issue go away rather than in this PR